tests/test_controller_output_handler.py

```
@@ -10,6 +10,10 @@
 from machine_common_sense.controller_output_handler import (
     ControllerOutputHandler, SceneEvent)
 from machine_common_sense.goal_metadata import GoalMetadata
+from machine_common_sense.logging_config import LoggingConfig
+
+# ignore printing of errors for unit testing
+LoggingConfig.init_logging(LoggingConfig.get_no_logging_config())
```

KeyError occurred in one of the tests but we don't need that printed out to the console so turning off logging (i.e. correctly swallow errors)